modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableRowSkin.java

```diff
@@ -172,6 +175,7 @@ private void setupTreeTableViewListeners() {
                 }
                 return null;
             }
+            // fall through
```

Comment:
There is no "fall through" here as we return from all conditions in previous case.

Reply:
there is a fall through (return is inside an `if` on line 173). 
The original code is a bit confusing, it took me a while to see the fall through cases, that's why I added explicit comments.

Reply:
There is a `return null` on line 176.
In that case, we will return from either line 173 or line 176. Hence, there is no fall through.

Reply:
Any comments on this?

Reply:
I am clearly in the wrong, sorry.  There is only one possible fall through for SELECTED_ITEMS -> CELL_AT_ROW_COLUMN which is not correct.

Q is, what should it return - a null or an empty list?  (I am in favor of a null because selection is impossible, as opposed to "nothing is selected".

Whether the attribute can be null should be specified in javafx.scene.AccessibleAttribute, do you think?

Reply:
And thank you @kleopatra for reference and reminder about test.
Out of this comes [JDK-8296413](https://bugs.openjdk.org/browse/JDK-8296413)
TableView with null focus model throws NPE in queryAccessibleAttribute(AccessibleAttribute.FOCUS_ITEM)